src/main/webapp/app/modules/conflicts/shared/components/location/single-location-details.tsx

```diff
@@ -45,6 +45,14 @@ export class SingleLocationDetails extends React.Component<ISingleLocationDetail
       .replace(/0\./, '')}`
   };
 
+  static getSelectOption = (record, locationNumber) => ({
+    value: locationNumber,
+    label:
+      record.physicalAddress !== null
+        ? `${record.physicalAddress.address1} ${record.physicalAddress.city}`
+        : `${record.postalAddress.address1} ${record.postalAddress.city}`
```

Comment:
What in case if both physicaladdress and postaladdress will be null? Is this possible?

Reply:
> What in case if both physicaladdress and postaladdress will be null? Is this possible?

yes, it's possible. I'll display location name in such case